src/common/config_opts.h

```diff
@@ -610,6 +610,7 @@ OPTION(osd_crush_chooseleaf_type, OPT_INT, 1) // 1 = host
 // This parameter is not consumed by ceph C code but the upstart scripts.
 // OPTION(osd_crush_initial_weight, OPT_DOUBLE, 0) // the initial weight is for newly added osds.
```

Comment:
can also remove this comment. it was added to document the the behaviour in  src/ceph-osd-prestart.sh